CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v2.11.0-rc.0 / 2019-07-03
+
+* [CHANGE] Remove max_retries from queue_config. #5649
+* [FEATURE] Add $externalLabels for alert unit tests. #5608
```

Comment:
Not quite, users can now provide external_labels

I think this and the next are enhancements, as they're building on existing features rather than being something brand new